src/helpers.ts

```diff
@@ -127,3 +127,38 @@ export async function pullRequestsForWorkflowRun(
 
   return pullRequests
 }
+
+export function squashCommit(
+  isSquashCommit: boolean,
+  squashTitle: boolean,
+  squashCommitTitle: string | null,
+  squashCommitMessage: string | null,
+  pullRequest: PullRequest,
+): {title: string | undefined, message: string | undefined} {
+  if (!isSquashCommit) {
+    return {title: undefined, message: undefined}
+  }
+
+  if (squashTitle) {
```

Comment:
I think we should remove this. Either way this is a breaking change given the new inputs have default values.

Reply:
Or maybe remove the defaults, since these pretty much mirror the GitHub defaults anyways.